docs/resources/virtual_environment_vm.md

```diff
@@ -11,6 +11,8 @@ subcategory: Virtual Environment
 
 Manages a virtual machine.
 
+> This resource uses SSH access to the node. You might need to configure the [`ssh` option in the `provider` section](../index.md#node-ip-address-used-for-ssh-connection).
+
```

Comment:
I'd probably remove quotation (`>`)
or add `[!NOTE]` header if you want to emphasize the text:

```
> [!NOTE]
> this is a note
```
rendered as 

> [!NOTE]
> this is a note

Reply:
Hmmm.... Terraform seems to be using a [different notation](https://github.com/hashicorp/terraform-provider-aws/blob/fddeae1cbbbe719d3799cecb91a8c8579b80f8b8/website/docs/r/api_gateway_account.html.markdown?plain=1#L13) for this type of blocks (rendered [example](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_account))

Reply:
I used the terraform notation now.